### PR TITLE
Feature/artifactory file

### PR DESCRIPTION
**Pull request by glow-mdsol**
@ncarroll-mdsol @jtomson-mdsol  for your consideration, it allows the Pod install to work with AppCode

### CODE REVIEW

File: AppConnectSwift/Babbage.podspec [hunk 1]
```diff
@@ -1,5 +1,12 @@
-username = ENV['ARTIFACTORY_USERNAME'] || raise("You must set an artifactory username using the environment variable ARTIFACTORY_USERNAME.")
-password = ENV['ARTIFACTORY_PASSWORD'] || raise("You must set an artifactory password using the environment variable ARTIFACTORY_PASSWORD.")
+if File.exist?('local.yaml')
+    require 'yaml'
+    content = YAML.load(File.open('local.yaml'))
+    username = content['ARTIFACTORY_USERNAME'] || raise("You must set an artifactory username using the variable ARTIFACTORY_USERNAME.")
```
Comment (jtomson-mdsol) [inline AppConnectSwift/Babbage.podspec]: might help to mention in the raised error that it's looking for this variable in the local.yaml file specifically?

File: AppConnectSwift/Babbage.podspec [hunk 2]
```diff
@@ -1,5 +1,12 @@
-username = ENV['ARTIFACTORY_USERNAME'] || raise("You must set an artifactory username using the environment variable ARTIFACTORY_USERNAME.")
-password = ENV['ARTIFACTORY_PASSWORD'] || raise("You must set an artifactory password using the environment variable ARTIFACTORY_PASSWORD.")
+if File.exist?('local.yaml')
+    require 'yaml'
+    content = YAML.load(File.open('local.yaml'))
```
Comment (ncarroll-mdsol) [inline AppConnectSwift/Babbage.podspec]: Nitpick: There's a `Yaml.load_file(...)` method that could shorten this a bit.

**Comment by ncarroll-mdsol** [comment]
Awesome, thank you @glow-mdsol! We do this in the appconnect-android app using the local.properties file, so it'll be nice to have that parity.

File: AppConnectSwift/Babbage.podspec (hunk 1 above)
Comment (glow-mdsol) [inline AppConnectSwift/Babbage.podspec]: Ok, updated

File: AppConnectSwift/Babbage.podspec (hunk 2 above)
Comment (glow-mdsol) [inline AppConnectSwift/Babbage.podspec]: Great, my ruby could use work.  Updated

**Comment by jtomson-mdsol** [comment]
Thanks geoff!

**Pull request merged**